ReflectometryServer/server_status_manager.py

```diff
@@ -193,6 +194,15 @@ def update_error_log(self, message: str, exception: Optional[Exception] = None):
             logger.exception(message, exc_info=exception)
         else:
             logger.error(message)
+
+        # If adding the new message exceeds the maximum character size of the error log,
+        # the oldest message is removed until the log size is under the maximum.
+        # One is added or removed from the size to compensate for the new line (check "_error_log_as_string").
+        MAX_ERROR_LOG_SIZE = 10_000
```

Comment:
is it possible to read this from the variable definition rather than hard code it?